src/logic/contracts/generateBatchRequests.ts

```diff
@@ -13,7 +13,7 @@ import { getWeb3 } from 'src/logic/wallets/getWeb3'
  */
 const generateBatchRequests = ({ abi, address, batch, context, methods }: any): any => {
   const web3 = getWeb3()
-  const contractInstance = new web3.eth.Contract(abi, address)
+  const contractInstance: any = new web3.eth.Contract(abi, address)
```

Comment:
Is it possible to correctly type this? I see that there is an interface `ContractInstance` in web3.js

Or maybe this can be helpful https://github.com/ethereum-ts/TypeChain